src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
```

Comment:
move it to * selector

Reply:
> move it to * selector

a lot mentors said don't use it and it's mistake